pkg/api/v1/types.go

```diff
@@ -1475,6 +1475,27 @@ type PodLogOptions struct {
 	Previous bool `json:"previous,omitempty" description:"return previous terminated container logs; defaults to false"`
 }
 
+// PodAttachOptions is the query options to a Pod's remote attach call
+// TODO: merge w/ PodExecOptions below for stdin, stdout, etc
```

Comment:
I agree, I'm open to other solutions, but my Go-fu was incapable of coming up with a better solution that didn't break the API (we could add an inline substruct, it will add a bunch of complexity in a variety of places (e.g. conversion))